portfolio/pom.xml

```diff
@@ -16,7 +16,8 @@
 
     <!-- Project-specific properties -->
     <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
-    <googleCloudProjectId>YOUR_PROJECT_ID_HERE</googleCloudProjectId>
+    <googleCloudProjectId>ndavar-sps-summer22
+</googleCloudProjectId>
```

Comment:
Why did you disrupt the whitespace and add an additional newline? `</googleCloudProjectId>` no longer is intended consistently.